package.json

```diff
@@ -11,21 +11,20 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "chai": "^4.2.0",
-    "dotenv": "^8.0.0",
+    "dotenv": "^8.2.0",
     "eslint-plugin-gettext": "^1.1.0",
-    "mocha": "^6.1.4",
-    "moment": "^2.24.0",
     "moment-range": "^4.0.2",
     "moment-timezone": "^0.5.28",
     "node-fetch": "^2.6.0",
     "parse-csv": "^0.2.0",
     "parse-link-header": "^1.0.1"
   },
   "devDependencies": {
+    "chai": "^4.2.0",
```

Comment:
thanks for moving chai and mocha to devDependencies.
I'll try to be better at remembering to put things there.